management/client.go

```diff
@@ -232,14 +232,160 @@ type OIDCBackchannelLogout struct {
 
 // ClientAddons defines the `addons` settings for a Client.
 type ClientAddons struct {
+	AWS                  *AWSClientAddon                  `json:"aws,omitempty"`
+	AzureBlob            *AzureBlobClientAddon            `json:"azure_blob,omitempty"`
+	AzureSB              *AzureSBClientAddon              `json:"azure_sb,omitempty"`
+	Box                  *BoxClientAddon                  `json:"box,omitempty"`
+	CloudBees            *CloudBeesClientAddon            `json:"cloudbees,omitempty"`
+	Concur               *ConcurClientAddon               `json:"concur,omitempty"`
+	Dropbox              *DropboxClientAddon              `json:"dropbox,omitempty"`
+	EchoSign             *EchoSignClientAddon             `json:"echosign,omitempty"`
+	Egnyte               *EgnyteClientAddon               `json:"egnyte,omitempty"`
+	Firebase             *FirebaseClientAddon             `json:"firebase,omitempty"`
+	Layer                *LayerClientAddon                `json:"layer,omitempty"`
+	MSCRM                *MSCRMClientAddon                `json:"mscrm,omitempty"`
+	NewRelic             *NewRelicClientAddon             `json:"newrelic,omitempty"`
+	Office365            *Office365ClientAddon            `json:"office365,omitempty"`
+	RMS                  *RMSClientAddon                  `json:"rms,omitempty"`
+	Salesforce           *SalesforceClientAddon           `json:"salesforce,omitempty"`
+	SalesforceAPI        *SalesforceAPIClientAddon        `json:"salesforce_api,omitempty"`
+	SalesforceSandboxAPI *SalesforceSandboxAPIClientAddon `json:"salesforce_sandbox_api,omitempty"`
+	SAML2                *SAML2ClientAddon                `json:"samlp,omitempty"`
+	SAPAPI               *SAPAPIClientAddon               `json:"sap_api,omitempty"`
+	Sharepoint           *SharepointClientAddon           `json:"sharepoint,omitempty"`
+	Sentry               *SentryClientAddon               `json:"sentry,omitempty"`
+	Slack                *SlackClientAddon                `json:"slack,omitempty"`
+	SpringCM             *SpringCMClientAddon             `json:"springcm,omitempty"`
+	SSOIntegration       *SSOntegrationClientAddon        `json:"sso_integration,omitempty"`
+	WAMS                 *WAMSClientAddon                 `json:"wams,omitempty"`
 	// SAML2 Addon configuration. Set this property to indicate that the Addon should be enabled, all settings are optional.
 	// The first entry in `Callbacks` should be the URL to post the SAML Token to.
```

Comment:
Should this comment be moved above SAML2?

Reply:
Comment moved